src/filter.rs

```diff
@@ -69,6 +69,36 @@ impl From<Level> for TracingFilter {
     }
 }
 
+/// Reads the tracing level configuration from environment variables.
+///
+/// This function checks the `OTEL_LOG_LEVEL` environment variable to determine
+/// the minimum tracing level. If the variable is not set or contains an invalid
+/// value, it defaults to [`Level::INFO`].
+///
+/// # Supported Values
+///
+/// The environment variable should contain one of:
+/// - `ERROR` or `error`
+/// - `WARN` or `warn`  
+/// - `INFO` or `info`
+/// - `DEBUG` or `debug`
+/// - `TRACE` or `trace`
+///
+/// # Returns
+///
+/// The configured [`Level`] for tracing, or [`Level::INFO`] as default
+///
+/// # Examples
+///
+/// ```bash
+/// export OTEL_LOG_LEVEL=DEBUG
+/// ```
+///
+/// ```rust
+/// use telemetry_rust::read_tracing_level_from_env;
+///
+/// let level = read_tracing_level_from_env();
```

Comment:
We shouldn't re-export it, but we could do this instead:

```suggestion
/// use telemetry_rust::otlp::read_otel_log_level_from_env;
///
/// let level = read_otel_log_level_from_env();
```